lib/pdk/tests/unit.rb

```diff
@@ -21,6 +23,32 @@ def self.invoke(tests, report = nil)
         output = PDK::CLI::Exec.execute(cmd(tests))
         report.write(output) if report
       end
+
+      def self.list
```

Comment:
A general stylistic suggestion for this method: Split the code up into logical paragraphs with a newline between them (e.g. paragraph 1 - bundler stuff, paragraph 2 - building the command array, paragraph 3 - executing the command, paragraph 4 - processing the output), makes it easier to grok a chunk at a time :)